More helpful README (#14830)
vercel/next.js: packages/create-next-app/README.md

# Create Next App

> The easiest way to get started with Next.js is by using `create-next-app`. This simple CLI tool enables you to quickly start building a new Next.js application, with everything set up for you. You can quickly create a new app using the default Next.js template, or choose from one of the <a aria-label="Next.js Starter Templates" href="https://github.com/vercel/next.js/tree/canary/examples">templates available here.</a> To get started, you can use the following command:

Reviewer: ```suggestion
The easiest way to get started with Next.js is by using `create-next-app`. This simple CLI tool enables you to quickly start building a new Next.js application, with everything set up for you. You can create a new app using the default Next.js template, or by using one of the [official Next.js examples](https://github.com/vercel/next.js/tree/canary/examples). To get started, use the following command:
```

# Create Next App

The easiest way to get started with Next.js is by using `create-next-app`. This simple CLI tool enables you to quickly start building a new Next.js application, with everything set up for you. You can quickly create a new app using the default Next.js template, or choose from one of the <a aria-label="Next.js Starter Templates" href="https://github.com/vercel/next.js/tree/canary/examples">templates available here.</a> 

> The following command will start the interactive experience for using `create-next-app`. The interactive experience will walk you through the process of creating the new app, and you can choose from the available templates directly from the command line:

Reviewer: This is quite redundant with the text that's above 🤔

Author: You are right!

> ```

Reviewer: ```suggestion
```bash
```

npx create-next-app
```

> To create a new app in a specified folder, you can start the command with the name of your new application. For example, the following command will create a new Next.js app called `blog-app` in a subdirectory called `blog-app`. 

Reviewer: ```suggestion
To create a new app in a specific folder, you can send a name as an argument. For example, the following command will create a new Next.js app called `blog-app` in a folder with the same name:
```

> ```

Reviewer: ```suggestion
```bash
```

npx create-next-app blog-app
```

## Options

`create-next-app` comes with the following options:

> * **-e, --example [name]|[github-url]** You can specify a template to use when creating a new Next.js app. You can specify a name of an example from the [official Next.js repository](https://github.com/vercel/next.js/tree/master/examples), or directly link to the template using this command. Example: `npx create-next-app -e with-passport`. This will then ask for the name of your app, and create a new Next.js app using the [`with-passport`](https://github.com/vercel/next.js/tree/master/examples/with-passport) example.

Reviewer: ```suggestion
* **-e, --example [name]|[github-url]** - An example to bootstrap the app with. You can use an example name from the [Next.js repo](https://github.com/vercel/next.js/tree/master/examples) or a GitHub URL. The URL can use any branch and/or subdirectory.
```

> * **--example-path &lt;path-to-example&gt;** In a rare case, your GitHub URL might contain a branch name with a slash (e.g. bug/fix-1) and the path to the example (e.g. foo/bar). In this case, you must specify the path to the example separately: `--example-path foo/bar`

Reviewer: ```suggestion
* **--example-path &lt;path-to-example&gt;** - In a rare case, your GitHub URL might contain a branch name with a slash (e.g. bug/fix-1) and the path to the example (e.g. foo/bar). In this case, you must specify the path to the example separately: `--example-path foo/bar`
```

## Why use Create Next App?

> `create-next-app` is the easiest way to get started with Next.js. The CLI tool enables you to create a new Next.js app within minutes. It is officially maintained by the creators of Next.js, and includes a number of benefits:

Reviewer: We already mention the beginning of this sentence at the start of the document

```suggestion
`create-next-app` allows you to create a new Next.js app within seconds. It is officially maintained by the creators of Next.js, and includes a number of benefits:
```

> * **Interactive Experience**: Running `npx create-next-app` (with no arguments) launches an interactive experience that guides you through setting up a project.

Reviewer: ```suggestion
* **Interactive Experience**: Running `npx create-next-app` launches an interactive experience that guides you through setting up a project.
```
